tests/infrastructure/instance_types/supported_os/test_rhel_os.py

```diff
@@ -57,6 +57,7 @@ def test_vmi_guest_agent_exists(self, golden_image_vm_with_instance_type):
         )
 
     @pytest.mark.polarion("CNV-11714")
+    @pytest.mark.special_infra
```

Comment:
migration tests should not need fully capable BM clusters. We run migration on PSI (as a part of smoke).

Reply:
ack